### ISSUE TITLE
Support multiple arch

### ISSUE THREAD

**Issue by FinweVI** [messagebird/sachet#58]
Hello,

it would be nice to provide builds for multiple arch (like ARMv7, for ARM Dev Board) for Sachet.
Providing a checksum of the files would be great too!

I can help with that if you're open to PR (:

**Comment by davidhitchins** [comment]
Totally agree! Is it possible, even now, to install on a Raspberry pi armv7?

**Comment by FinweVI** [comment]
Yes you can, but you'll have to build the binary from the source yourself.

**Comment by davidhitchins** [comment]
I'd love to try but I am not a coder. However, I can work these things out if given a push in the right direction on how to do things. Any suggestion for a starting point (webpage, article etc) would be most welcome. :-)
